### DISCUSSION TITLE
Two tabs to two panes in one tab?

### DISCUSSION THREAD

**Discussion by animalsThink** [wezterm/wezterm#2043] [Q&A] [answered]
Ex. You have w3m on one tab and vim on the other. Now you want to have them side by side as two panes in one tab.
Is there any way to do this?

**Comment by wez** [comment, marked as answer]
We don't yet have a fully-developed set of functionality around this, but in the nightly build as of c4a248f1a3fac4a3c30e8849f893ae33a6ea1a85 you may now use `wezterm cli split-pane --move-pane-id PANE` to move `PANE` from its current tab, create a split in the active pane and put `PANE` there.

You can use `wezterm cli list` to get a list of panes and their ids, or `echo $WEZTERM_PANE` from inside one of the panes to see its pane id.

**Reply by animalsThink** [reply]
Hi, wez
Thank you for your efforts. I love your wezterm.
As of the new feature, it doesn't work. Or maybe I failed to do things right.
I build wezterm copy -and-pasting the instructions on 1.5 Build from source page.
 
cargo run --release --bin wezterm -- start
w3m
open a new tab
cd Downloads
vim
go back to w3m tab
ctrl-z (w3m in bg job)
wezterm cli list
wezterm cli split-pane --move-pane-id 1  <-- failed
wezterm cli split-pane --move-pane-id 0  <-- failed

Best regards
![2022-05-28-163627_1920x1080_scrot](https://user-images.githubusercontent.com/106414369/170817148-37461f78-a71d-48b7-b9f5-681eb66e0993.png)
![2022-05-28-163803_1920x1080_scrot](https://user-images.githubusercontent.com/106414369/170817167-44567636-937d-4c4c-a8c7-11802c22bf5c.png)

**Reply by animalsThink** [reply]
Sorry. I misunderstood how to do the trick.
The new feature worked fine. Thank you, wez.
I didn't placed the newly built binary in $PATH and had used an older wezterm.
